python/py-dulwich/Portfile

```diff
@@ -18,26 +18,30 @@ long_description    Simple Pure-Python implementation of the Git file \
 
 homepage            https://www.dulwich.io
 
-checksums           rmd160  c81b9d7f4b6a65b4699b4f2e046fe471aa4dc29f \
-                    sha256  08b7e0ad62c06c82810ac91925512d2a6ae9875625fc67e8d50150b089bd0a94 \
-                    size    363564
+checksums           rmd160  3557712ed0c45d73c104a5dd5dc215bfb3e940cb \
+                    sha256  2b2512ad80e03a1d9ffaafc38d884992b40ac7cfa6ddd814db098733365712dc \
+                    size    370540
 
-python.versions     27
+python.versions     27 38
 
 if {${name} ne ${subport}} {
-    conflicts           py${python.version}-dulwich-devel
-    if {[string match "*-devel" $subport]} {
-        conflicts           py${python.version}-dulwich
+    # 0.20 dropped support for Python 2.7
+    if {${python.version} == 27} {
+        github.setup            dulwich dulwich 0.19.16 dulwich-
+        github.livecheck.regex  {(0.1[0-9.]+)}
+
+        checksums               rmd160  3557712ed0c45d73c104a5dd5dc215bfb3e940cb \
```

Comment:
these checksums are not right (cannot be the same as above for version 0.20.2) - the builds fail because of this

Reply:
I've been using `port bump` to set or update these. Looks like I'll have to be a bit more critical of what it does…